src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -393,33 +399,65 @@ private static String emptyStringArrayDefaultValue(Node defaultValueNode,
     private static String getNodeText(Node node) {
         final StringBuilder result = new StringBuilder(20);
 
+        boolean skipExampleSection = false;
+        boolean alreadySkippedExamplesSection = false;
```

Comment:
Because in `visitClass` we are asserting that
```java
      .that(getJavaDocText(node))
      .isEqualTo(expected);
```

`getJavaDocText()` calls `getNodeText()`. We skip the examples section in both `getJavaDocText()` and `expected` and compare them.

Or do you mean there is a better way to do this that I overlooked?

Reply:
I think I need to look at the code closer.

Reply:
@rnveach I removed the Javadoc examples and reverted this code here.